bindings/flow/FDBLoanerTypes.h

```diff
@@ -205,7 +205,7 @@ namespace FDB {
 	};
 
 	struct KeyRangeRef {
-		const KeyRef begin, end;
+		KeyRef begin, end;
```

Comment:
I don’t like the idea of just removing the const here. Instead I would suggest you do also make these fields private and write a public const getter. That way the semantics of the class doesn’t change. Otherwise users might think they can change the values.

Reply:
I started working on this, but it seems like it will be an enormous change that will be very hard to merge. There are many hundreds of usages of the `begin` and `end` fields.

Reply:
It may be worthwhile though, as having `KeyRangeRef` be const-correct could avoid a lot worse problems in the future.

Reply:
Maybe we could do it in a separate PR?

Reply:
Yes, sadly flatbuffers doesn't play well with const correctness because we create an instance of the object before we deserialize. I think a getter would still be worth the effort. Alternatively it might be possible to create a new type `KeyRef` that is read-only except by flatbuffers. This could potentially touch less code.

If you want to do this in a separate PR I would suggest to roll back this change and keep it `const` for now.